src/StagesCards.js

```diff
@@ -1,14 +1,41 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './styles/App.css';
 
 
-
-
-
-
-
-
-
-
+class StagesCards extends Component {
+
+  render() {
+    if(this.props.stage === '') {
```

Comment:
Empty strings are falsey, so you should be able to just check `if (!this.props.stage)`